profile/README.md

```diff
@@ -0,0 +1,91 @@
+## Welcome to the world of Zarhus - Your Digital Chess Knight 👋
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/zarhus/.github/main/profile/img/zarhus-logo.png" width="500" align=center/>
+</p>
+
+Inspired by the elegance and strategic prowess of the "huszár", meaning knight
+in chess (Polish: skoczek), Zarhus brings a unique blend of tradition and
```

Comment:
@artur-rs we could also point from where originally the word huszar comes from

Reply:
@TomaszAIR improved: 58a50e2d08ac55c6c925429d44be00192086e55f